master/getting-started/openstack/upgrade/upgrade.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Upgrading Calico 
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/openstack/upgrade/upgrade'
+---
+
+## Calico package update
+
+This part of the upgrade procedure varies slightly according to which OS you are using.  
+
+- [Upgrading an OpenStack cluster based on CentOS](#upgrading-an-openstack-cluster-based-on-centos)
+
+- [Upgrading an OpenStack cluster based on Ubuntu](#upgrading-an-openstack-cluster-based-on-ubuntu)
+
+> **Important**: Do not use older versions of `calicoctl` after the upgrade.
+> This may result in unexpected behavior and data.
+{: .alert .alert-danger}
+
+
+## Upgrading an OpenStack cluster based on CentOS
+   
+1. On all nodes, change the location of the Calico packages to point to the 3.x repo:
+
+   ```
+   sudo sed -i 's/calico-2.6/calico-3.1/g' /etc/yum.repos.d/calico.repo 
+   ```
+
+1. Upgrade the compute software
+   
```

Comment:
I don't think you need to use nested lists for this. Not sure they add anything.... I think you can just delete `1. Upgrade the compute software` and `1. Upgrade the Control software:` and move the nested lists to the left for a single level.